### ISSUE TITLE
intermittent door responses

### ISSUE THREAD

**Issue by scotmerrick** [ratgdo/mqtt-ratgdo#87]
Have 3 ratgdos and they work well for the most part.  More frequently, the open or close commands do not cause motion of the doors.  Seems like after power failure or upgrades to my Universal Devices Polisy plugin controller (Goose66 author), I lose door status  ("unknown") and the commands fail to open or close door.

**Comment by Goose66** [comment]
What "mode" (dry contact, security+ 2.0, security+ 1)?

**Comment by scotmerrick** [comment]
The Ratgdo boards use v2.57 security +1.0, 2.0 & dry contactSHMOn May 6, 2024, at 1:17 PM, W. Randy King ***@***.***> wrote:﻿
What "mode" (dry contact, security+ 2.0, security+ 1)?

**Comment by Goose66** [comment]
No, you said you had 3 ratgdos controlling three garage door openers that are having the problem. What mode(s) are you using with your three garage door openers.

**Comment by scotmerrick** [comment]
Sorry. What are you asking for?SHMOn May 6, 2024, at 6:37 PM, Scot Merrick ***@***.***> wrote:﻿All are the sameSHMOn May 6, 2024, at 5:55 PM, W. Randy King ***@***.***> wrote:﻿
No, you said you had 3 ratgdos controlling three garage door openers that are having the problem. What mode(s) are you using with your three garage door openers.

**Comment by Goose66** [comment]
Let's see if I can make this clearer: when you setup your ratgdo for your garage door openers, what operation mode did you select ("sec+ 2.0", "sec+ 1", or "dry contact")?

**Comment by scotmerrick** [comment]
The firmware is 2.57 and the card is setup as “dry contact”

 

SHM

 

From: "W. Randy King" ***@***.***>
Reply-To: ratgdo/mqtt-ratgdo ***@***.***>
Date: Monday, May 6, 2024 at 9:15 PM
To: ratgdo/mqtt-ratgdo ***@***.***>
Cc: Merrick Mill Valley ***@***.***>, Author ***@***.***>
Subject: Re: [ratgdo/mqtt-ratgdo] intermittent door responses (Issue #87)

 

Let's see if I can make this clearer: when you setup your ratgdo for your garage door openers, what operation mode did you select ("sec+ 2.0", "sec+ 1", or "dry contact")?
